test/puppetlabs/puppetdb/scf/storage_test.clj

```diff
@@ -1130,6 +1130,14 @@
        (assoc report
          :puppet_version "3.2.1 (Puppet Enterprise 3.0.0-preview0-168-g32c839e)") timestamp)))
 
+  (deftest report-with-event-timestamp
+    (let [z-report (update-in report [:resource_events] (fn [events]
```

Comment:
This `update-in` could probably go in a function in the let with a name that makes it a little more clear what it's doing.
```clojure
[add-event-timetstamps (fn [report timestamp]
                                        (update-in report [:resource_events] (partial map #(assoc % :timestamp timestamp))))]
```